Resources/Prototypes/DeltaV/Entities/Structures/Storage/SecureCabinet/secure_cabinets.yml

```diff
@@ -0,0 +1,318 @@
+#Parent
+- type: entity
+  abstract: true
+  parent: [ BaseStructureDynamic, BaseBagOpenClose ]
+  id: BaseSecureCabinet
+  suffix: Empty
+  description: A secure cabinet to keep all your confidential files or items away from prying hands.
+  categories: [ HideSpawnMenu ]
+  components:
+  - type: Storage
+    grid:
+    - 0,0,2,1
+    - 0,3,2,4
+    - 0,6,2,7
+    maxItemSize: Normal
+  - type: Sprite
+    sprite: DeltaV/Structures/Storage/secure_cabinet.rsi
+    noRot: true
+  - type: Appearance
+  - type: UserInterface
+    interfaces:
+      enum.StorageUiKey.Key:
+        type: StorageBoundUserInterface
+  - type: Transform
+    noRot: true
+  - type: Fixtures
+    fixtures:
+      fix1:
+        shape:
+          !type:PhysShapeAabb
+          bounds: "-0.22,-0.40,0.22,0.40"
+        density: 200
+        mask:
+        - MachineMask
+        layer:
+        - MachineLayer
+  - type: InteractionOutline
+  - type: ContainerContainer
+    containers:
+      storagebase: !type:Container
+  - type: Damageable
+    damageContainer: StructuralInorganic
+    damageModifierSet: Metallic
+  - type: Destructible
+    thresholds:
+    - trigger:
+        !type:DamageTrigger
+        damage: 50
+      behaviors:
+      - !type:DoActsBehavior
+        acts: ["Destruction"]
+      - !type:PlaySoundBehavior
+        sound:
+          collection: MetalBreak
+      - !type:SpawnEntitiesBehavior
+        spawn:
+          SheetSteel1:
+            min: 1
+            max: 2
+  - type: StaticPrice
+    price: 80
+  - type: Construction
+    graph: SecureCabinet
+    node: securecabinet
+  
+#Empty Secure Cabinet
+- type: entity
+  parent: BaseSecureCabinet
+  id: SecureCabinet
+  name: Secure Cabinet
+  suffix: Empty
+  components:
+  - type: Lock
+  - type: LockVisuals
+  - type: AccessReader
+  - type: Sprite
+    state: secure-cabinet
+    layers:
+    - state: secure-cabinet
+    - state: secure-cabinet-open
+      map: ["openLayer"]
+    - state: unlocked
+      shader: unshaded
+    - state: locked
+      map: ["enum.LockVisualLayers.Lock"]
+      shader: unshaded
+
+#Filled Secure Cabinet
+- type: entity
+  parent: SecureCabinet
+  id: FilledSecureCabinet
+  suffix: Filled
+  categories: [ HideSpawnMenu ]
+  components:
+  - type: StorageFill
+    contents:
+      - id: Pen
+        prob: 0.5
+      - id: PaperOffice
+        amount: 1
+        maxAmount: 2
+      - id: Paper
+        amount: 1
+        maxAmount: 2
+      - id: BoxFolderBlue
+        orGroup: Folder
+      - id: BoxFolderRed
+        orGroup: Folder
+      - id: BoxFolderYellow
+        orGroup: Folder
+      - id: BoxFolderWhite
+        orGroup: Folder
+      - id: BoxFolderGrey
+        orGroup: Folder
+      - id: BoxFolderBlack
+        orGroup: Folder
```

Comment:
use `EntityTableContainerFill` on container `storagebase` instead of `StorageFill`

Reply:
So then, I have addressed almost all of this, except for this one.
Could you explain what you want to do here exactly? 
Should I remove lines 96 to 116 and replace StorageFill with EntityTableContainerFill?
And if so, do I need to add/edit a file to define what is in the cabinets or not?
I am no coder, so I have been mostly working off of what I have found in other yml files in this repository.

Reply:
you can do it in the same file by *not* using a nested selector:
```yml
containers:
  storagebase: !type:AllSelector
    children:
    - id: Pen
      prob: 0.5
    - id: PaperOffice
      amount: !type:RangeNumberSelector
        range: 1, 2
    - id: Paper
      amount: !type:RangeNumberSelector
        range: 1, 2
    - !type:GroupSelector
      children:
      - id: BoxFolderBlue
      - id: BoxFolderRed
      - id: BoxFolderYellow
      - id: BoxFolderWhite
      - id: BoxFolderGrey
      - id: BoxFolderBlack
```